ios/FluentUI/Button/Button.swift

```diff
@@ -133,12 +134,14 @@ open class Button: UIButton, TokenizedControlInternal {
     }
 
     open override func didUpdateFocus(in context: UIFocusUpdateContext, with coordinator: UIFocusAnimationCoordinator) {
-        guard style == .accent || style == .danger,
```

Comment:
Is it ok that we're doing this for controls that are not just accent and danger?